src/components/EntryEditor/EntryEditor.js

```diff
@@ -39,9 +46,66 @@ class EntryEditor extends Component {
         onRemoveAsset,
         onCancelEdit,
     } = this.props;
+    const controlClassName = `${ styles.controlPane } ${ this.state.showEventBlocker && styles.blocker }`;
+    return (
```

Comment:
I wouldn't copy all the markup when the only changes are wrapping with the SplitPane and removing the preview.